### PR TITLE
Chore/merge main@4ffdef3

### PR DESCRIPTION
**Pull request by julien-deramond**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    attributes:
+      label: Describe the issue
+      description: Provide a summary of the issue and what you expected to happen, including specific steps to reproduce.
+    validations:
+      required: true
+  - type: textarea
+    id: reduced-test-case
+    attributes:
+      label: Reduced test cases
+      description: Include links [reduced test case](https://css-tricks.com/reduced-test-cases/) links or suggested fixes using [CodePen](https://codepen.io/).
```
Comment (julien-deramond) [inline .github/ISSUE_TEMPLATE/bug_report.yml:29]: We don't have a [CodePen Teams](https://codepen.io/features/teams) account so we can't do the same thing than Bootstrap: `using CodePen ([v4 template](https://codepen.io/team/bootstrap/pen/yLabNQL) or [v5 template](https://codepen.io/team/bootstrap/pen/qBamdLj)).` which is nice. When this PR will merged, I will track this as an issue and see if we could have a [CodePen Teams](https://codepen.io/features/teams).

**Comment by julien-deramond** [comment]
Got a Netlify error:
```
7:23:18 AM: /opt/build/repo/node_modules/postcss-cli/index.js:3
7:23:18 AM: import fs from 'fs-extra'
7:23:18 AM:        ^^
7:23:18 AM: SyntaxError: Unexpected identifier
7:23:18 AM:     at Module._compile (internal/modules/cjs/loader.js:723:23)
7:23:18 AM:     at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
7:23:18 AM:     at Module.load (internal/modules/cjs/loader.js:653:32)
7:23:18 AM:     at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
7:23:18 AM:     at Function.Module._load (internal/modules/cjs/loader.js:585:3)
7:23:18 AM:     at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
7:23:18 AM:     at startup (internal/bootstrap/node.js:283:19)
7:23:18 AM:     at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
7:23:18 AM: /opt/build/repo/node_modules/postcss-cli/index.js:3
7:23:18 AM: import fs from 'fs-extra'
7:23:18 AM:        ^^
7:23:18 AM: SyntaxError: Unexpected identifier
7:23:18 AM:     at Module._compile (internal/modules/cjs/loader.js:723:23)
7:23:18 AM:     at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
7:23:18 AM:     at Module.load (internal/modules/cjs/loader.js:653:32)
7:23:18 AM:     at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
7:23:18 AM:     at Function.Module._load (internal/modules/cjs/loader.js:585:3)
7:23:18 AM:     at Function.Module.runMain (internal/modules/cjs/loader.js:831:12)
7:23:18 AM:     at startup (internal/bootstrap/node.js:283:19)
7:23:18 AM:     at bootstrapNodeJSCore (internal/bootstrap/node.js:623:3)
7:23:18 AM: npm ERR! code ELIFECYCLE
7:23:18 AM: npmnpm ERR! errno 1
7:23:18 AM:  ERR! code ELIFECYCLE
7:23:18 AM: npm ERR! errno 1
7:23:18 AM: npm ERR! boosted@5.1.1 css-prefix-main: `postcss --config build/postcss.config.js --replace "dist/css/*.css" "!dist/css/*.rtl*.css" "!dist/css/*.min.css"`
```
Not sure why for the moment.

File: scss/_variables.scss [hunk 2]
```diff
@@ -1056,20 +1056,23 @@ $form-check-inline-margin-end:    1rem !default;
 // scss-docs-end form-check-variables
 
 // scss-docs-start form-switch-variables
-$form-switch-width:               $spacer * 3 !default;
+// Boosted mod: no $form-switch-color
+$form-switch-width:               $spacer * 3 !default; // Boosted mod
```
Comment (Nurovek) [inline scss/_variables.scss:1060]: I'm not sure about the boosted mod here ?

Comment (Lausselloic) [reply scss/_variables.scss:1060]: https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss#L928 2em in bootstrap

**Review by Nurovek — APPROVED** [review]

**Pull request merged**